doc/ApiDocGeneration/Generate-Api-Docs.ps1

```diff
@@ -182,4 +182,13 @@ $mutatedContent = $mutatedContent -replace $hrefRegex, '"./$1"'
 Set-Content -Path $baseUrl -Value $mutatedContent -NoNewline
 
 Write-Verbose "Compress and copy HTML into the staging Area"
+Write-Verbose "Path to Archive"
```

Comment:
@benbp , @chidozieononiwu , @weshaggard Could you please help review the change to this file. It LGTM but want to make sure you get a chance to review as you likely have more context here.

Reply:
Changes look fine to me as well. @sima-zhu do you have any thoughts?

Reply:
The reason why we failed this is because in our `Build and Package` step, we create the folder over there. 
<img width="848" alt="image" src="https://user-images.githubusercontent.com/48036328/180567875-c9852fb8-512d-491e-b6d3-0761727951f1.png">

There is no source code for `Azure.Analytics.Purview.Share`, so the `Generate-API-Docs.ps1` failed to upload docs. 
For artifacts, we usually contain both source package and docs package instead of having docs only. That's why we do not create folder if not exist. Could you understand why the share package does not show on build step?

Expected to see (something like storage):
<img width="392" alt="image" src="https://user-images.githubusercontent.com/48036328/180568592-21212707-9f5e-4069-9f1d-c028b39a5b7c.png">

Actually seen:
<img width="315" alt="image" src="https://user-images.githubusercontent.com/48036328/180568671-69e19721-f019-442c-90f4-bd15c8f010f6.png">

Reply:
Thank you @sima-zhu , as discussed over teams, the name was incorrect, it was Azure.Analytics.Share which was supposed to be Azure.Analytics.Purview.Share, that was fixed, the powershell scripts is undone now. The pipeline passed as the correct folder name exists.